src/qt/bitcoingui.cpp

```diff
@@ -364,6 +365,9 @@ void BitcoinGUI::createActions()
         connect(openAction, SIGNAL(triggered()), this, SLOT(openClicked()));
     }
 #endif // ENABLE_WALLET
+
+    new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_C), this, SLOT(showDebugWindowActivateConsole()));
```

Comment:
This looks weird, allocating a freely-floating object, but I do think it is correct. The object will become a child of `*this`, and automatically freed with it.